src/collections/posts/curator-applications-are-now-open.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Curator applications are currently open!
+section: announcements
+category: announcement
+publish: 2024-04-07T18:37:19.983Z
+image: /static/beastsaber-logo-fullsize-square.jpg
+showInPostListing: false
+---
+# Curation Recruitment is Currently Open!
+
+**We are re-opening our curator applications! We’re looking for 3 outstanding applicants of any skill level. These applicants should be people who:**
```

Comment:
The ' on we're seems to be the "fancy" one again - if you wanted to change that.

Reply:
~~Maybe I just like my fancy apostrophes!~~
I will update it